packages/app/components/stack-modal/StackProgress.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  OrderProps,
+  fundsUsedWithToken,
+  totalOrdersDone,
+  totalFundsUsed,
+} from "@/app/models/order";
+import { OrdersProgressBar } from "@/components/OrdersProgressBar";
+import { BodyText } from "@/ui";
+
+export const StackProgress = ({ order }: OrderProps) => (
+  <div className="mt-3 space-y-3">
+    <div className="flex flex-col justify-between space-y-1 md:space-y-0 md:items-center md:flex-row">
+      <OrdersExecuted order={order} />
+      <div className="flex items-center space-x-1">
+        <BodyText size="responsive" className="text-em-low">
+          Total funds used:{" "}
+          <span className="text-em-high">
+            {totalFundsUsed(order).toFixed(2)}{" "}
+            <span className="text-xs text-em-low">of</span>{" "}
+            {fundsUsedWithToken(order)}
+          </span>
```

Comment:
This looks a bit different from the design, any reason in particular?

Reply:
was that the difference you noticed?

Reply:
yeah, this part is getting big and the "of" is not important, so I made it smaller and decreased the emphasis.

<img width="410" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/2547b5c7-7c5b-4e3a-ad64-d6ff9f59277a">